undebt/examples/deprecated_inequality_operator.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from pyparsing import Word
+
+from undebt.pattern.lang.python import UNARY_OP_ATOM
+from undebt.pattern.util import tokens_as_list
+
+
+grammar = UNARY_OP_ATOM + Word("<>").suppress() + UNARY_OP_ATOM
```

Comment:
`UNARY_OP_ATOM` is actually necessary, since `EXPR` will actually match `<>` as a valid operator.